src/pages/Offering.tsx

```diff
@@ -10,95 +10,84 @@ import { Legal } from '../sections/offering/legal'
 import { PriceInfo } from '../sections/offering/price-info'
 import { TopBar } from '../sections/offering/top-bar'
 
-export const Container = styled('div')({})
+export const OuterContainer = styled('div')({})
```

Comment:
Imo we should rename this to something more clear. `Container` is kinda claimed by `constate` once we pulled that in. Maybe `Wrappper`?